keycloak/themes/hmda/login/messages/messages_en.properties

```diff
@@ -24,3 +24,7 @@ invalidPasswordNotUsernameMessage=&#8226; Invalid password: must not be equal to
 invalidPasswordRegexPatternMessage=&#8226; Invalid password: fails to match regex pattern(s).
 invalidPasswordHistoryMessage=&#8226; Invalid password: must not be equal to any of last {0} passwords.
 invalidPasswordGenericMessage=&#8226; Invalid password: new password doesn''t match password policies
+
+# custom for hmda
+# DO NOT COMMIT UPDATES TO OR REMOVE THE "support@localhost.localdomain"
+hmdaEnterEmailAddress=To register as a filer for more than one institution, you may select multiple institutions from the list of available institutions based on your email domain. If one or more institutions is not available in the list based on your email domain, contact support@localhost.localdomain for assistance with registering for the additional institution(s).
```

Comment:
Updated in https://github.com/cfpb/hmda-platform-auth/pull/112/commits/0ecbade8e266f1e03003859af4375ae297611683